src/cmd/cli/command/commands.go

```diff
@@ -618,6 +621,11 @@ var composeCmd = &cobra.Command{
 	Short:   "Work with local Compose files",
 }
 
+func printPortalURL() {
+	cli.Info(" * Monitor your services' status in the defang portal")
```

Comment:
Do you think it's useful to show the portal URL per service? For example, https://portal.defang.dev/service/service1 etc.. This could be more useful especially if you are deploying a bunch of service separately, then you don't have to figure out which one you just deployed.